StocksDashboard.py

```diff
@@ -15,7 +14,7 @@
 HEIGHT = 648
 
 
-__all__ = ['Base_StocksDashboard', 'StocksDashboard', 'datetime', 'get_colors']
+__all__ = ['StocksDashboard', 'datetime', 'get_colors']
 
 
 def datetime(x):
```

Comment:
This name can be confused with standard datetime object (https://docs.python.org/2/library/datetime.html)

Reply:
ok, I took it from this example in bokeh [stocks.py](https://bokeh.pydata.org/en/latest/docs/gallery/stocks.html). I'll add it as a static method then.